workers.tf

```diff
@@ -338,6 +338,28 @@ resource "aws_security_group_rule" "workers_ingress_cluster_https" {
   type                     = "ingress"
 }
 
+resource "aws_security_group_rule" "workers_ingress_cluster_primary" {
+  count                    = var.worker_create_security_group && var.worker_create_cluster_primary_security_group_rules && local.cluster_primary_security_group_id != "" && var.create_eks ? 1 : 0
```

Comment:
These count arguments do not work on an empty state. `local.cluster_primary_security_group_id != ""` is unknown so count cannot be calculated. Replace it with `var.cluster_version >= 1.14`.

```
Error: Invalid count argument

  on ../../workers.tf line 353, in resource "aws_security_group_rule" "cluster_primary_ingress_workers":
 353:   count                    = var.worker_create_security_group && var.worker_create_cluster_primary_security_group_rules && local.cluster_primary_security_group_id != "" && var.create_eks ? 1 : 0

The "count" value depends on resource attributes that cannot be determined
until apply, so Terraform cannot predict how many instances will be created.
To work around this, use the -target argument to first apply only the
resources that the count depends on.
```

Reply:
This line also needs updating